classes/form/config.php

```diff
@@ -111,7 +111,7 @@ public function definition() {
         $mform->setType("prodwwwroot", PARAM_URL);
         $mform->setDefault("prodwwwroot", envbarlib::getprodwwwroot());
 
-        if ($CFG->allowmultipledomains) {
+        if ($CFG->allowmultipledomains ?? false) {
```

Comment:
would  it be cleaner to write it like `if (!empty($CFG->allowmultipledomains)) `  see https://www.php.net/manual/en/function.empty.php